bgpd/bgp_nht.c

```diff
@@ -454,6 +421,77 @@ void bgp_parse_nexthop_update(int command, vrf_id_t vrf_id)
 	evaluate_paths(bnc);
 }
 
+void bgp_parse_nexthop_update(int command, vrf_id_t vrf_id)
```

Comment:
This part *sure* seems like it's own separate commit and not part of this change right?

Reply:
Agreed. The NHT-related changes were separated into their own commit.